Class-18-(Angular-Lesson-1).md

```diff
@@ -0,0 +1,36 @@
+## In the prep work this week (Angular Lesson 1), the students learned:
+
+1. The concept of *templates* and *components* and why we want to use them to build websites.
+1. How to use the Angular CLI to create a new project.
+1. The Angular folder structure generated by ``ng new project-name.``
+1. How to generate Angular components via the command line.
+1. How to display data in the view by adding placeholders to the HTML file.
+1. How to use ``ng serve`` to preview an Angular project locally in a browser.
+
+## Large Group Time (Instructor Notes)
+
+### Announcements
+
+1. Graded assignment #6 is available. Encourage students to begin working on it this week if they have not already started.
+1. Check with your class Candidate Engagement Manager for any other announcements.
+
+### Topics That Require More Attention
+
+1. Angular is a big leap forward in terms of new skills. There is a LOT of content packed into the last three lessons. Be prepared to offer reassurance.
+1. Q & A as needed for the prep-work exercises.
+1. Reinforce the benefits for using templates and components.
+    1. Pull up the LaunchCode website (launchcode.org) and use the dev tools to examine how many different parts make up the page.
+    1. What features do the different pages of the website have in common?
+    1. How do templates help us? (Class discussion).
+1. Review the Angular CLI commands for creating a new project and generating new components.
+1. Can the HTML tag defined for a nested component be used directly in the ``app.component.html`` file? Explore with the students to see.
+1. Studio intro.
+
+## Studio (TF Notes)
+
+1. If you do not know Angular well, you must ABSOLUTELY complete ALL of the prep work yourself! We expect the Angular lessons to be a big lift, and your students need you to be in top form.
```

Comment:
I would rephrase this: "If you do not know Angular well, you need to do the prep work! The Angular lessons are a big lift and students really need your support and technical prowess to succeed."

Reply:
Done.  Thanks.